db/schema.rb

```diff
@@ -20,11 +20,18 @@
   enable_extension "uuid-ossp"
 
   create_table "kudos", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
-    t.text     "body",        null: false
+    t.text     "body",       null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.uuid     "giver_id",   null: false
+  end
+
+  create_table "kudos_to_receivers", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
     t.datetime "created_at",  null: false
     t.datetime "updated_at",  null: false
-    t.uuid     "giver_id",    null: false
+    t.uuid     "kudo_id",     null: false
     t.uuid     "receiver_id", null: false
+    t.index ["kudo_id", "receiver_id"], name: "index_kudos_to_receivers_on_kudo_id_and_receiver_id", unique: true, using: :btree
```

Comment:
Should we add another index on `receiver_id`?

Reply:
Yeah, good idea!